### ISSUE TITLE
Feature Request: More Search Results Per Page - Magneticow

### ISSUE THREAD

**Issue by Maverik44** [boramalper/magnetico#101]
I am counting #14 results per page. It would be helpful to allow more results.

#14, 28, 42, 56, 70

**Comment by boramalper** [comment]
There are 20 results per page, not 14. =)

**Comment by Maverik44** [comment]
Yup I realized that one last night also. I tired to change the value from 20 to 40.

**Comment by mweinelt** [comment]
#131 exposes the limit as a GET parameter, it's missing a proper UI component. I have no idea how to implement that properly, so if anyone wants to take up that quest, please go ahead.
